frontend/src/utils/index.js

```diff
@@ -543,15 +543,55 @@ export function addKymaAddon (options) {
   shootAddonList.push(kymaAddon)
 }
 
-export function compileMarkdown (text) {
+export function textColorFromColor (color) {
+  const iteratee = value => /^(darken|lighten|accent)-\d$/.test(value) ? 'text--' + value : value + '--text'
+  return map(split(color, ' '), iteratee)
+}
+
+function htmlToDocumentFragment (html) {
+  var template = document.createElement('template')
+  html = html.trim() // Never return a text node of whitespace as the result
+  template.innerHTML = html
+  return template.content
+}
+
+function documentFragmentToHtml (documentFragment) {
+  const div = document.createElement('div')
+  div.appendChild(documentFragment.cloneNode(true))
+  return div.innerHTML
+}
+
+export function compileMarkdown (text, linkColor = 'cyan darken-2', transformToExternalLinks = true) {
   if (!text) {
     return undefined
   }
-  return DOMPurify.sanitize(marked(text, {
+  let html = DOMPurify.sanitize(marked(text, {
     gfm: true,
     breaks: true,
     tables: true
   }))
+
+  const textColorClasses = textColorFromColor(linkColor)
+  console.log(textColorClasses)
+  const documentFragment = htmlToDocumentFragment(html)
+  if (documentFragment) {
```

Comment:
early exit

Reply:
yes I refactored this and also the last part makes no sense (`documentFragmentToHtml(documentFragment`)
Will fix...

Reply:
done